SKYKit.podspec

```diff
@@ -34,4 +34,11 @@ Pod::Spec.new do |s|
     facebook.dependency 'SKYKit/Core'
     facebook.dependency 'FBSDKCoreKit', '~> 4.0'
   end
+
+  s.subspec 'ForgotPassword' do |forgotPassword|
```

Comment:
* The `default_subspec` should be a new subspec called `Default` (something like that)
  * If user install `SKYKit` it is the same as installing `SKYKit/Default`
* The `Default` subspec should include `Facebook` and `ForgotPassword` subspecs as dependency.

This should be easy. Let me know if there is a problem.